crates/store/re_log_types/src/path/entity_path_filter.rs

```diff
@@ -128,6 +128,12 @@ impl std::iter::Sum for EntityPathFilter {
     }
 }
 
+impl<S: AsRef<str>> From<S> for EntityPathFilter {
+    fn from(rules: S) -> Self {
+        Self::parse_forgiving(rules.as_ref(), &EntityPathSubs::default())
+    }
+}
```

Comment:
I see the utility here in test code, though worry about losing semantic annotation that this is a "forgiving" parse rather than a conversion.

I wish we had a strict parse and go do this with a `TryFrom` instead:
```
"/**".try_into()?
```
would actually be nice for test code to avoid accidentally ending up with malformed entity path expressions.

That said, I seems unlikely to matter in practice.

Reply:
that's a good point. I think most of these happen in test, so it would be perfectly valid to have `unwrap` or `?`. I'll make that change (unless something really blocking shows up).

Reply:
Mmm. This is actually slightly more involved, but I think very much worth doing. I'll capture it in an issue and add a TODO.

Reply:
I'm gonna merge because I'm blocked on it—let me know if you see something to be fixed in a follow-up PR.